stats/influxdb/config.go

```diff
@@ -49,13 +49,19 @@ type Config struct {
 	Retention    null.String `json:"retention,omitempty" envconfig:"K6_INFLUXDB_RETENTION"`
 	Consistency  null.String `json:"consistency,omitempty" envconfig:"K6_INFLUXDB_CONSISTENCY"`
 	TagsAsFields []string    `json:"tagsAsFields,omitempty" envconfig:"K6_INFLUXDB_TAGS_AS_FIELDS"`
+	BoolFields   []string    `json:"boolFields,omitempty" envconfig:"K6_INFLUXDB_BOOL_FIELDS"`
+	FloatFields  []string    `json:"floatFields,omitempty" envconfig:"K6_INFLUXDB_FLOAT_FIELDS"`
+	IntFields    []string    `json:"intFields,omitempty" envconfig:"K6_INFLUXDB_INT_FIELDS"`
 }
 
 func NewConfig() *Config {
 	c := &Config{
 		Addr:             null.NewString("http://localhost:8086", false),
 		DB:               null.NewString("k6", false),
 		TagsAsFields:     []string{"vu", "iter", "url"},
+		BoolFields:       []string{},
+		FloatFields:      []string{},
+		IntFields:        []string{},
```

Comment:
I do think that both `vu` and `iter` are perfect candidates for IntFields ;)

Reply:
Once a field is created, InfluxDB does not allow to change the type.  I think we need to keep default (string) as it is today for backward compatibility...

Reply:
does it convert it back or does it drop the new values ?

I would argue this is probably how it should've worked and will be helpful either way - I don't know how many people use it,but they can always set it the previous value or decide that this is worthy to loose some historic data. 
But all new users will basically not have to worry about it

Reply:
I'm not sure if we should change this, as @TamiTakamiya says, the types can't be changed after the first insertion.

Otherwise InfluxDB will drop stats with this error:

```
ERRO[0000] InfluxDB: Couldn't write stats                error="{\"error\":\"partial write: field type conflict: input field \\\"mybooltag\\\" on measurement \\\"http_reqs\\\" is type boolean, already exists as type string dropped=8\"}\n"
```

This is also an issue if you specify a field in `K6_INFLUXDB_TAGS_AS_FIELDS`, but don't specify it in e.g. `K6_INFLUXDB_BOOL_FIELDS`. The first time the field is inserted it will be a string, and any subsequent insertions will fail. You would have to use a different field name, [use some shard workaround](https://docs.influxdata.com/influxdb/v1.8/troubleshooting/frequently-asked-questions/#can-i-change-a-field-s-data-type), or drop the DB. :-/

In any case, users should be able to manually cast `vu` and `iter` to `integer` in their queries [if they need it](https://docs.influxdata.com/influxdb/v1.8/query_language/explore-data/#cast-operations) (assuming string -> int casting works...).